source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -291,8 +359,44 @@ content: |
              status: 0,
              masterKey: { provider: 'local' }
            }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           from pprint import pprint
+           connection_string = "mongodb://localhost:27017"
+           key_vault_namespace = "encryption.__keyVault"
+
+
+           def verify_data_key_created(fle_opts, client, client_encryption,  data_key):
```

Comment:
Looks like there's an extra space in here. Could you run the linter on this snippet, just in case?

Reply:
Two more nitpicks: 
1) The parameter order here would make more sense with the important arguments first. Since you are performing a "verify" on a "data key", perhaps `data_key` could be first, then `client` and then `client_encryption` & `fle_opts` (next to each other since they are related).
2) The side-effect of closing clients is unexpected by a helper function that does not create the new client. Usually, you would expect that in the scope of the function that constructed the client. Since no clients are being created in this snippet, can we omit lines 377-378?

Reply:
Fixed:
* Ran Linter
* Fixed parameter order per your spec ^
* Removed closing of the client

Reply:
Did you check in the fixes mentioned above? For some reason, I do not see the changes yet.

Reply:
Re-pushed